lib/rails_icons/icon.rb

```diff
@@ -66,7 +68,7 @@ def default_stroke_width
   end
 
   def library_set_attributes
-    return custom_library || {} if custom_library?
+    return custom_library.dig(set) || {} if custom_library?
```

Comment:
I updated the `custom_library` method implementation. Initially the method was returning the digged `set` value. But now it returns library.
```ruby
&.dig(@library) || {}
```

And I dig in the returned hash by `custom_library` method in `library_set_attributes` method. But I think that might not work.

I was thinking instead of updating both `custom_library` and `library_set_attributes`, I can just do this.

```ruby
&.dig(*set ? [@library, set] : [@library]) || {}
```

Reply:
so far its working